gta/altv/egg-pterodactyl-alt--v.json

```diff
@@ -13,7 +13,7 @@
         "ALT:V": "ghcr.io/parkervcp/games:altv"
     },
     "file_denylist": [],
-    "startup": "./altv-server",
+    "startup": "./altv-server --port {{SERVER_PORT}}",
```

Comment:
its allready in config parsing? Why is that needed?

`"port:": "port: {{server.build.default.port}}",`